### PR TITLE
[15.0][MIG] purchase_force_invoiced: Migration to 15.0

### PR DESCRIPTION
**Pull request by AnnaPForgeFlow**
@ForgeFlow
Standard migration to 15.0

### CODE REVIEW

File: purchase_force_invoiced/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* Jordi Ballester <jordi.ballester@eficent.com>
```
Comment (MiquelRForgeFlow) [inline purchase_force_invoiced/readme/CONTRIBUTORS.rst]:
```suggestion
* Jordi Ballester <jordi.ballester@forgeflow.com>
```

File: purchase_force_invoiced/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
```
Comment (MiquelRForgeFlow) [inline purchase_force_invoiced/__manifest__.py]: -> ForgeFlow

**Comment by pedrobaeza** [comment]
This is already migrated in #1485

**Comment by sergio-teruel** [comment]
Can you please include #1531 ?
There is other PR openend https://github.com/OCA/purchase-workflow/pull/1485 but @Olageibol does not answer

**Comment by MiquelRForgeFlow** [comment]
@sergio-teruel PR is ready. I would close the other old PR.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Comment by sergio-teruel** [comment]
Thanks @MiquelRForgeFlow

**Pull request merged**